lib/modules/download/utils.rb

```diff
@@ -80,6 +80,7 @@ def self.filters_dump filters
     end
 
     def self.search_token term, filters
+      return 'all' if (term.empty? && filters.empty?)
```

Comment:
Do you know if we have access to the `geo_type` query param in this method? If so, instead of just passing `all` we could pass `all-#{geo_type.pluralise}` where it could be all countries, all regions etc. which would be more informative